### PR TITLE
First release☺️, but too many job have done

### PR DESCRIPTION
**Pull request by soc221b**

### CODE REVIEW

File: scripts/test.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ echo '''
                                       Nose
 ================================================================================
 '''
-nosetests . -v --with-coverage --cover-erase
```
Comment (soc221b) [inline scripts/test.sh:23]: Generally, we always need to know which line is covered or else in local environment.

File: src/enums.py [hunk 2]
```diff
@@ -37,12 +37,15 @@
...
 
-FORMATS = {'letter_case': ['upper_case',
-                           'lower_case',
-                           'camel_case',
-                           'pascal_case'],
-
-           'sep': ['dash_to_underscore',
-                   'underscore_to_dash'],
-
-           'abbr': ['remain', 'convert']}
```
Comment (soc221b) [inline src/enums.py:48]: Removed abbreviation supports, this will add in future, but not now.

File: src/file_checker.py [hunk 3]
```diff
@@ -1,70 +1,272 @@
 import os
 import re
-from enums import FORMATS
-from settings import (CHECK_DIRS, FILE_FORMATS, DIR_FORMATS,
-                      IGNORE_FILES, IGNORE_DIRS)
```
Comment (soc221b) [inline src/file_checker.py:5]: Separated main and file_checker files.

File: src/settings.py [hunk 4]
```diff
@@ -1,15 +0,0 @@
...
-IGNORE_FILES = [r".*\.md$"]
-
-IGNORE_DIRS = [os.sep.join([r'.*', r'unclassified$']),
-               os.sep.join([r'.*', r'unclassified', r'.*$'])]
-
-FILE_FORMATS = {'letter_case': 'lower_case',
-                'sep': 'dash_to_underscore',
-                'abbr': 'convert'}
-
-DIR_FORMATS = {}
```
Comment (soc221b) [inline src/settings.py:15]: Use json, remove compile dependency and increate user-friendly.

**Comment by arnavb** [comment]
Whoo 🎆 😄 

Before you merge this, however, don't forget to add a `setup.py`!

**Comment by soc221b** [comment]
@arnavb Ok, I will create another PR for this and `setup.py` because these commits are chaotic.

**Pull request closed without merging**